source/_components/doorbird.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: "DoorBird"
+description: "Instructions on how to integrate your DoorBird video doorbell with Home Assistant."
+date: 2017-08-06 11:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: doorbird.png
+ha_category: Hub
+ha_release: "0.50"
```

Comment:
Release will need to be updated before merge. Most likely to 0.52 since 0.51 is rather imminent.